docs/upgrade.md

```diff
@@ -79,21 +79,26 @@ istio-sidecar-injector-596f8dddbb-gvzk9   1/1       Running   0          1m
 istio-telemetry-7cbf75c5cf-wk4v8          2/2       Running   0          1m
 ```
 
-The `Istio` Custom Resource is showing `Available` in its status field and the Istio components are using `1.4.5` images :
+The `Istio` Custom Resource is showing `Available` in its status field and the Istio components are using `1.4.8` images :
 
 ```bash
-$ kubectl describe istio -n istio-system istio -o yaml | grep "image:"
```

Comment:
kubectl describe istio -n istio-system istio -o yaml | grep "image:"  this is a bug